3.5.0: TestByteRange fails on ppc64el

~~~~
=== RUN   TestByteRange
--- FAIL: TestByteRange (0.00s)
    lock_test.go:84: unexpected error while locking file /tmp/byterange-742630381: invalid argument
FAIL
FAIL	github.com/sylabs/singularity/pkg/util/fs/lock	1.007s
~~~~

See full build log [here](https://buildd.debian.org/status/fetch.php?pkg=singularity-container&arch=ppc64el&ver=3.5.0%2Bds1-1&stamp=1573712708&raw=0).

Could it be that the test just don't wait long enough?

https://github.com/sylabs/singularity/blob/master/pkg/util/fs/lock/lock_test.go#L48

--- a/pkg/util/fs/lock/const.go
+++ b/pkg/util/fs/lock/const.go
@@ -3,8 +3,6 @@
 // LICENSE file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-// +build !linux
-
 package lock
 
 import "golang.org/x/sys/unix"

--- a/pkg/util/fs/lock/const_linux_32bit.go
+++ b/pkg/util/fs/lock/const_linux_32bit.go
@@ -3,6 +3,8 @@
 // LICENSE file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
+// +build linux,386 linux,arm linux,mips linux,mipsle
+
 package lock
 
 import "golang.org/x/sys/unix"

